run.py

```diff
@@ -577,7 +598,7 @@ def run_replanning_loop(settings, forecast_year):
             forecast_land_use(settings, year, forecast_year, client)
 
         else:
-            forecast_year = year
+            forecast_year = start_year
```

Comment:
this doesn't seem right. on the first time through, `year == start_year`, but if we want to iterate asim + beam multiple times, `forecast_year` won't get updated.

Reply:
So I was running into a problem before making this change where, when running with usim turned off, after the first year asim would look in the urbansim datastore for e.g. 2011 data and crash when it didn't find anything. I guess we could also pass start year to activitysim if urbansim is turned off?

Reply:
Hmm, so in the use case you're describing, you want ActivitySim to read from the original input .h5 over and over rather than the last iteration of ActiviySim outputs?

Reply:
I'll re-run to confirm that this is still an issue, but I guess the problem was that not all of the needed land use data got copied over from the original input .h5 into the last iteration of ActivitySim outputs, so activitysim throws a file missing error. So maybe the fix then is to make sure everything is copied over rather than to change this year. I'll take a closer look

Reply:
Ah, I see. Yeah, OK. If we aren't running UrbanSim, then the forecast year _should_ be the start_year, because you're really only specifying a start/end year for the sake of iteration, not because you're actually forecasting outputs for a future year. Is that right?

Reply:
Right, exactly. What I didn't think through is that this could mean that not all of the previous-iteration asim outputs might not get correctly read into the next asim run. So what I still haven't figured out is whether this means we need to include a _another_ counter to keep track of which asim iteration we're on, or we should copy the rest of the usim outputs, or whether it's working OK as is